server/src/rooms/game/state/index.ts

```diff
@@ -1153,11 +1158,13 @@ export class GameState extends Schema implements GameData {
   // REFACTOR: DEFAULT CATEGORY
 
   log(
+    round: number,
```

Comment:
I'd suggest we remove this in a later refactor (or now if there's time). The `GameState` already has the round number implicit in it, so you could just instantiate the MarsLogMessage later with `this.round`

Reply:
resolved in 5b58247